packages/node_modules/@ciscospark/redux-module-spaces/src/actions.js

```diff
@@ -121,10 +119,20 @@ export function getSpaces(sparkInstance, options = {}) {
               return Promise.resolve(new Error('Space was not decrypted correctly'));
             });
         }
+        else {
+          // Since we are not decrypting, save the redux store and do one big push
+          users.push(...space.participants.items);
+          activities.push(...space.activities.items);
+        }
 
         return Object.assign({}, space, {isDecrypting: deferDecrypt});
       });
 
+      const uniqueUsers = uniqBy(users, 'id');
```

Comment:
Users could be in multiple spaces, but we only need one instance per user in our store.